### PR TITLE
[docs]: Fix documentation for the SVG Image Component

### PR DESCRIPTION
**Pull request by sanketshinde3001**
Solved the issue - #10898 

Changes done in both docs (current and Version 2.50.0 (LTS) )

@RudraPratik30 Solved that Bold Text issue.

### CODE REVIEW

File: docs/docs/widgets/svg-image.md [hunk 1]
```diff
@@ -26,51 +26,51 @@ Refer to the resources where SVG files are downloaded for free. You copy-paste t
...
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Layout
 
 | <div style={{ width:"100px"}}> Layout </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div> |
 |:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/svg-image.md:63]: Hey, can you just add the full stops at the end? In the expected value column, probably the last change needed.
Please make the changes in both versions.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

**Review by RudraPratik30 — APPROVED** [review]
Hey, @sanketshinde3001 approving the PR from my end and forwarding it to the team for final review.
Thanks for contributing!
Your PR will be merged by the end of this week

**Comment by RudraPratik30** [comment]
Hey @sanketshinde3001 can you please sign the [Contributor License Agreement](https://cla-assistant.io/ToolJet/ToolJet?pullRequest=10910) before we can accept your contribution.

**Comment by sanketshinde3001** [comment]
@RudraPratik30 Done Sir 👍

**Comment by sanketshinde3001** [comment]
@RudraPratik30 waiting to merge this pr.

**Comment by RudraPratik30** [comment]
Hey @sanketshinde3001, sorry the things got delayed since the CLA was not signed, but we assure you that your PR will be merged as soon as possible, you can expect it by the end of this week.

**Comment by sanketshinde3001** [comment]
@RudraPratik30  Ohk Sir. I signed CLA already.

**Pull request merged**